Nos/Models/ReplyCount.swift

```diff
@@ -0,0 +1,8 @@
+/// Kind of counter (used when displaying a Note)
+enum ReplyCount {
```

Comment:
I'd like to propose a different name for this enum but I'm struggling. `Count` only applies to one of the cases, so a broader term would be nice. `RepliesDisplayType`, maybe?

Reply:
Renamed to RepliesDisplayType